code/game/objects/weapons.dm

```diff
@@ -1,4 +1,11 @@
 /obj/item/weapon
 	name = "weapon"
 	icon = 'icons/obj/weapons.dmi'
-	hitsound = "swing_hit"
+
+/obj/item/weapon/New()
```

Comment:
This takes me way back to the days when not everything had a hitsound; stabbing someone with a scalpel was very effective because it was high damage AND silent.

Reply:
Cardboard sheets do (some)  damage and is silent 😇

Reply:
I'm not sure that things that specifically do not have a hit sound should be assigned a hit sound on New.